provisioning/miniprov/hare_mp/main.py

```diff
@@ -523,16 +523,31 @@ def generate_config(url: str, path_to_cdf: str) -> None:
     if path:
         path += os.pathsep + '/opt/seagate/cortx/hare/bin/'
     python_path = os.pathsep.join(sys.path)
+
+    conf = ConfStoreProvider(url)
```

Comment:
@mssawant the node-name file will be written by `configure` script (it repeats the logic of hare-bootstrap, so node-name file is populated with the name evaluated from consul-agents.json file).

But the point is that if you write to node-name here, its content will be overwritten below by `configure` script, so this code has no effect.

Reply:
Yes, agree. Abandoning this PR. will have to add consul import to configure.py or invoke update-consul-conf again.